android/filament-android/src/main/cpp/Renderer.cpp

```diff
@@ -58,8 +58,8 @@ Java_com_google_android_filament_Renderer_nMirrorFrame(JNIEnv *, jclass, jlong n
         jint flags) {
     Renderer *renderer = (Renderer *) nativeRenderer;
     SwapChain *dstSwapChain = (SwapChain *) nativeDstSwapChain;
-    const Viewport dstViewport {dstLeft, dstBottom, (uint32_t) dstWidth, (uint32_t) dstHeight};
-    const Viewport srcViewport {srcLeft, srcBottom, (uint32_t) srcWidth, (uint32_t) srcHeight};
+    const filament::Viewport dstViewport {dstLeft, dstBottom, (uint32_t) dstWidth, (uint32_t) dstHeight};
```

Comment:
Why add filament:: to Viewport and not the other types?

Reply:
It's just because there is an ambiguity in cpp files between driver::Viewport and filament::Viewport.

Reply:
Should we just rename driver::Viewport so we don't have to put filament:: everywhere like in your CL?